.bazelrc

```diff
@@ -0,0 +1,9 @@
+build --action_env CC=clang
+build --action_env CXX=clang++
+build --cxxopt=-std=c++14 
+build --host_cxxopt=-std=c++14 
+build --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0"
```

Comment:
Nit: be consistent about quoting flag values. Most of the above are not quoted but this and most below are.

Reply:
Done!